funnel/templates/project_cfp.html.jinja2

```diff
@@ -0,0 +1,90 @@
+  {% assets "css_jquerytimepicker" -%}
+    <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}" />
+  {%- endassets -%}
+{% endblock %}
+
+{% block content %}
+  {%- if form.errors %}
+    <div class="alert alert--error alert--dismissable">
+      <a class="alert__close" href="javascript:void(0);" aria-label="close"><i class="material-icons">clear</i></a>
+      <p class="alert__text"><i class="material-icons alert__icon">error</i> {% trans %}Please review the indicated issues{% endtrans %}</p>
+    </div>
+  {%- endif %}
+  {%- if message %}
+    <p class="form-message">{{ message }}</p>
+  {%- endif %}
+  <form data-parsley-validate="true" id="cfp-form" method="POST" action="{{ request.url }}" accept-charset="UTF-8" class="mui-form mui-form--margins hg-form">
+    {{ form.hidden_tag() }}
+    {{ renderfield(form.instructions, autofocus=true) }}
+    {%- for error in form.instructions.errors %}
+      <div><p class="mui-form__error">{{ error }}</p></div>
+    {%- endfor %}
+    {% if not form.cfp_start_at.data %}
+      <div id="cfp-publish-actions" class="btn-wrapper hide">
+        <button name="open-now" class="mui-btn cfp-action-btn">Open CFP now</button>
+      </div>
+    {% endif %}
+    {% if form.cfp_start_at.data %}
+      {{ renderfield(form.cfp_start_at, css_class="input-transition js-cfp-start-at") }}
+    {% else %}
+      {{ renderfield(form.cfp_start_at, css_class="input-transition js-cfp-start-at") }}
+    {% endif %}
+    {%- for error in form.cfp_start_at.errors %}
+      <div><p class="mui-form__error">{{ error }}</p></div>
+    {%- endfor %}
+    {% if form.cfp_end_at.data %}
+      {{ renderfield(form.cfp_end_at, css_class="input-transition js-cfp-end-at") }}
+    {% else %}
+       {{ renderfield(form.cfp_end_at, css_class="input-transition js-cfp-end-at") }}
+    {% endif %}
+    {%- for error in form.cfp_end_at.errors %}
+      <div><p class="mui-form__error">{{ error }}</p></div>
+    {%- endfor %}
+    <div class="mui-form form-actions clearfix">
+      <div>
+        <button type="submit" name="submit" class="mui-btn mui-btn--raised mui-btn--primary" value="reject">Submit</button>
+        <span class="loading mui--hide"><i class="material-icons mui--align-middle mui--text-dark-secondary">refresh</i></span>
+      </div>
+    </div>
+  </form>
+{% endblock %}
+
+{% block footerscripts %}
+  {% assets "js_timepicker" -%}
+    <script type="text/javascript" src="{{ ASSET_URL }}"></script>
+  {%- endassets -%}
+  <script type="text/javascript">
+    $(function() {
+      $('input.datetime-time').timepicker({ 'scrollDefaultNow': true, 'timeFormat': 'H:i:s' });
+      // Set current date & time in cfp_start_at input field and show cfp_end_at input field.
+      $('button[name="open-now"]').click(function(event) {
+        event.preventDefault();
+        $('#cfp_start_at-date').val(new Date().toISOString().split('T')[0]);
+        $('#cfp_start_at-time').val(new Date().toLocaleTimeString());
```

Comment:
The backend can't interpret locale-based fields. It should be in ISO 24 hours format, which means `.toISOString().split('T')[1]` (with the timezone suffix stripped out).